ext/opcache/zend_file_cache.c

```diff
@@ -1957,23 +1961,23 @@ zend_persistent_script *zend_file_cache_script_load(zend_file_handle *file_handl
 			goto use_process_mem;
 		}
 
-		buf = zend_shared_alloc_aligned(info.mem_size);
+		buf = zend_shared_alloc_aligned(cache_handle.info.mem_size);
 
 		if (!buf) {
 			zend_accel_schedule_restart_if_necessary(ACCEL_RESTART_OOM);
 			zend_shared_alloc_unlock();
 			goto use_process_mem;
 		}
-		memcpy(buf, mem, info.mem_size);
+		memcpy(buf, cache_handle.mem, cache_handle.info.mem_size);
 		zend_map_ptr_extend(ZCSG(map_ptr_last));
 	} else {
 use_process_mem:
-		buf = mem;
+		buf = cache_handle.mem;
 		cache_it = false;
 	}
 
-	ZCG(mem) = ((char*)mem + info.mem_size);
-	script = (zend_persistent_script*)((char*)buf + info.script_offset);
+	ZCG(mem) = ((char*)cache_handle.mem + cache_handle.info.mem_size);
```

Comment:
Probably nothing to worry about if we won't go with this method anyway, but, for my own learning experience, does this look correct?:

```c
ZCG(mem)
```

I'm not sure that feels right when the other variable is now `cache_handle.mem`, however, it compiles and runs fine.